.ci/Jenkinsfile

```diff
@@ -251,13 +251,20 @@ pipeline {
   }
 }
 
-def checkTestSuite(Map parallelTasks = [:], String suite, String tags) {
-  def regexps = [ "^e2e/_suites/${suite}/.*", "^.ci/.*", "^cli/.*", "^e2e/.*\\.go" ]
-  if ("${FORCE_SKIP_GIT_CHECKS}" == "true" || isGitRegionMatch(patterns: regexps, shouldMatchAll: false)) {
-    log(level: 'INFO', text: "Adding ${suite}:${tags} test suite to the build execution")
-    parallelTasks["${suite}_${tags}"] = generateFunctionalTestStep(suite: "${suite}", tags: "${tags}")
-  } else {
-    log(level: 'WARN', text: "The ${suite}:${tags} test suite won't be executed because there are no modified files")
+def checkTestSuite(Map parallelTasks = [:], Map item = [:]) {
+  def suite = item.suite
+  def platforms = suite.platforms
```

Comment:
https://beats-ci.elastic.co/blue/organizations/jenkins/e2e-tests%2Fe2e-testing-mbp/detail/PR-654/7/pipeline/279

CI is failing with:
>No such field found: field java.lang.String platforms

Any guess?

Reply:
is it a list? or maybe  `item.platforms` similar to `item.scenarios`

Reply:
It's exactly like `scenarios`: https://github.com/elastic/e2e-testing/pull/654/files#diff-b87fe24438639288bccba6a647eac526dec8e1755dcf1fc7959434a3ba95404aR4-R6

Reply:
My bad... I've just seen it... It's `item.platforms` instead of `suite.platforms`

Reply:
# #
![image](https://media.giphy.com/media/l0HlDHQEiIdY3kxlm/giphy.gif)